htdocs/config2/locale.inc.php

```diff
@@ -39,6 +39,12 @@
 $opt['template']['locales']['NL']['flag'] = 'images/flag/NL.png';
 $opt['template']['locales']['NL']['name'] = 'Nederlands';
 
+/*
+    Disabling unused locales speeds up template precompilation (clear-webcache.php).
+
+    for more examples please have a look in commit 526787528ce608a122060afaee0b830617c3bf53
+*/
+
```

Comment:
Disagree. The lines that you re-added are connected to the lines that you did not re-add. These are not "more examples" but the rest of the settings for the same languages.

All these are prepared settings which belong to the current OC code distribution.